labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -59,7 +59,7 @@ object ListManipulationExercise01 {
    *  - ... etc 
    */
   def concatLists[T](l1: List[T], l2: List[T]): List[T] = {
-    error("fix me")
+    l1.union(l2)
   }
 
```

Comment:
I'm actually not sure how union works, but my guess would be (based on the documentation) that it treats the lists as sets, and takes their set union, which isn't the same as concatenation. Its possible I'm wrong though. This can also be done just using "++".